TechTurtel.md

```diff
@@ -0,0 +1,32 @@
+---
+name: ["@TechTurtel", "@addu2468"]
+project: "3D Printed Kinetic & Digital Clock"
+---
+
+# Project Name
```

Comment:
I'm using this guide - https://www.instructables.com/Kinetic-Digital-Clock-Arduino-3D-Print , as a reference but I don't want to use their files, so I'm going to try and design my own first.

The budget wasn't too clear but its (3x10) in there, i'm going to be using 30 servos

For now I don't plan on adding a screen for the modules, I just want to display the values using the clock numbers. For example when I press the button it will switch to temperature and will display live temperature value on the numbers. Same thing for carbon monoxide, sound (decibels), light, and maybe other ones if I can add them.

Soldering station is mainly for the extra modules and I really want to get into soldering things

I kind of have an idea from looking at the guide, and I'm going to try prototyping myself, the gears should transfer the rotation of the servo into front and back movement.

Reply:
@TechTurtel great! thank you hehe :) could you please add to your file: 
- that you're referring to this guide
- you'll be using 30 servos

thank you sm! and absolutely love that you're going to try prototyping a lot of things yourself and will be bringing the project beyond what is on the guide :)